web/public/js/services/intercom-service.js

```diff
@@ -8,6 +8,11 @@ angular.module('cpZenPlatform').factory('intercomService', function ($localStora
     return _.contains(user.roles, 'cdf-admin');
   }
 
+  function userIsNinja(user) {
+    var initUserTypeStr = JSON.parse(user.initUserType);
```

Comment:
It is initialized inside [bootIntercom()](https://github.com/CoderDojo/cp-zen-platform/blob/enable-intercom-for-all-but-ninjas/web/public/js/services/intercom-service.js#L37), not outside function scope